### PR TITLE
[175]: Fix for spot bug

### LINKED ISSUE #175: spotbug error in hbase
### Software Environment:
- OpenLooKeng version (source or binary):

- OS platform & distribution (eg., Linux Ubuntu 16.04):

- Java version:
### Steps to reproduce the issue
1.
2.
3.

### Special notes for this issue

### PR DESCRIPTION
**Pull request by rajesh322**
### What type of PR is this?

Uncomment only one /kind <> line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug /kind task /kind feature

### Which issue(s) this PR fixes:

Fixes #175 

### Special notes for your reviewers:

### CODE REVIEW

File: hetu-hbase/src/test/java/io/hetu/core/plugin/hbase/metadata/TestingHBaseTableUtils.java [hunk 1]
```diff
@@ -32,7 +36,10 @@ private TestingHBaseTableUtils() {}
     public static void delFile(String file)
     {
         File pfile = new File(file);
-        pfile.delete();
+        boolean ret = pfile.delete();
+        if (!ret) {
```
Comment (sraghunandan) [inline hetu-hbase/src/test/java/io/hetu/core/plugin/hbase/metadata/TestingHBaseTableUtils.java:40]: will fail with checkstyle

**Comment by RajeevRastogi** [comment]
lgtm

File: hetu-hbase/src/test/java/io/hetu/core/plugin/hbase/metadata/TestingHBaseTableUtils.java (hunk 1 above)
Comment (sraghunandan) [inline hetu-hbase/src/test/java/io/hetu/core/plugin/hbase/metadata/TestingHBaseTableUtils.java:40]: Not required to take to a variable

Comment (rajesh322) [reply hetu-hbase/src/test/java/io/hetu/core/plugin/hbase/metadata/TestingHBaseTableUtils.java]: fixed

**Pull request merged**